src/app/services/table.service.ts

```diff
@@ -19,8 +19,8 @@ export class TableService {
     this.dataTable = new DataTable();
     this.prevRowDimensions = [];
     this.prevColDimensions = [];
-    this.mockDataUpdate();
-    this.getDimensions();
+    // todo: this.mockDataUpdate();
+    // todo: this.getDimensions();
```

Comment:
looks like an old version of the table branch, since it has been merged into dev, maybe directly rebase this PR with dev

Reply:
OK, will rebase on dev.